$NetBSD: patch-ac,v 1.5 2010/01/09 19:44:05 tron Exp $

Remove bogus cast which breaks the build with Sun Studio C.

--- net.c.orig	2008-09-19 10:37:57.000000000 +0100
+++ net.c	2010-01-09 19:34:47.000000000 +0000
@@ -1205,7 +1205,7 @@
   switch ( saddr->sa_family ) {
   case AF_INET:
     sa4 = (struct sockaddr_in *) saddr;
-    strncpy( strptr, inet_ntoa( (struct in_addr) sa4->sin_addr ), len - 1 );
+    strncpy( strptr, inet_ntoa( sa4->sin_addr ), len - 1 );
     strptr[ len - 1 ] = '\0';
     return;
 #ifdef ENABLE_IPV6
